### PR TITLE
connected to mongo db

### PR DESCRIPTION
**Pull request by Sisi-tech**
create a db folder with connect.js, and update server.js to connect to MongoDB.

### CODE REVIEW

File: src/models/user.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    },
+    lastName: {
+        type: String,
+        trim: true,
+        required: [true, 'Please provide last name'],
+        minlength: 1,
+        maxlength: 50,
+    },
+    email: {
+        type: String,
```
Comment (jgoldenberg29) [inline src/models/user.js:21]: lets add lowercase to ensure that all of our emails are in the same format.

**Review by jgoldenberg29 — CHANGES_REQUESTED** [review]
Code is looking really good Sisi! Great first pull request. I just suggested one change/addition for email in the user schema. Keep it up!

**Review by katsiarynalashcheuskaya — APPROVED** [review]
Great use of validation for `email`, `password`, and `schema indexes`! There’s just a small merge conflict to address. 

Adding `timestamps: true` to the schema could be an optional improvement for the future, as it would automatically include `createdAt` and `updatedAt` fields. These might be useful for tracking record changes. 

Great work overall, @Sisi-tech!

File: src/models/user.js (hunk 1 above)
Comment (Sisi-tech) [reply src/models/user.js:21]: I'll add lowercase to the emails

Comment (Sisi-tech) [reply src/models/user.js:21]: I added lowercase to the emails

**Review by jgoldenberg29 — APPROVED** [review]
Great job implementing those changes   👍

**Pull request closed without merging**